### ISSUE TITLE
Smooth in test, not in steam vr

### ISSUE THREAD

**Issue by SuperEvenSteven** [HipsterSloth/PSMoveSteamVRBridge#42]
I've got 4 cameras, I've distributed them evenly on my usb hubs and calibrated them well. I'm experiencing an issue with the open vr driver you've written. The PSMove client program renders the controllers very well but the PSMoveSsteamVRBridge is jittery and shows the controller flipping as soon as there's a little momentum. I have both the driver and the client program running and comparing the two is night and day. Do you have any suggestions on why this might be please? 

[Edit] I forgot to mention I experience this jittery flipping when I'm testing in the SteamVR app. Strangely though, if the app enters standby mode (geometric lines and cameras only) the controllers don't exhibit the behaviour. As soon as I move the headset and the SteamVR app starts again the controllers are back to being jittery.

**Comment by saadminhas** [comment]
I am facing the same issue.
Any Leads?

**Comment by machmar** [comment]
You don't have strong enough computer. I had the same issue until I bought a new pc. When you do something, the computing power isn't enough. Try checking the task manager for CPU usage and turn off some of the programs that eat a lot of CPU.

**Comment by SuperEvenSteven** [comment]
At the time I opened this issue I'd just upgraded my computer to a core i7 8700K 16GB ram and GTX 980. I found the best way to diagnose the cause is to use the PSMoveService config tool and go through each camera checking the bounding box doesn't flicker. I walked around the room for each camera and observed that the controller bulb was detected correctly at all times. Changes in room lighting conditions can sometimes reduce the accurracy of your original calibration.
